### ISSUE TITLE
id3v2.4 comments such as iTunPGAP & iTunSMPB in common.comments 

### ISSUE THREAD

**Issue by rossgrady** [Borewit/music-metadata#502]
I'm happy to try to submit a PR to correct this but I'm not yet 100% clear on where this would be caught in your codebase.

**Bug description**
common.comments array has length 4 & contains the contents of COMM.iTunPGAP, COMM.iTunSMPB and COMM.iTunNORM in addition to COMM

**Expected behavior**
common.comments will only contain human-readable comments from the "Comments" field

**Audio file demonstrating the problem**
If applicable, please drag and drop the __zipped__ audio file(s) which caused the problem into the bug report.
Please create additional issues if you experience different problems for each file.
[08 Billy.mp3.zip](https://github.com/Borewit/music-metadata/files/4568691/08.Billy.mp3.zip)

**Comment by Borewit** [comment]
I am very busy at the moment, please forgive if I don't reply timely next time. A PR would be great @rossgrady.

Would such a mapping, do the trick?: https://github.com/Borewit/music-metadata/blob/cb7ab1f9f2c640c6b3c8f121c90a45b636a28d5f/lib/mp4/MP4TagMapper.ts#L17

**Comment by rossgrady** [comment]
This is the chunk of the native object that contains the multiple COMM ids: 

```
native:
   { 'ID3v2.4':
      [ { id: 'TIT2', value: 'Billy' },
        { id: 'TPE1', value: 'Swamp Dogg' },
        { id: 'COMM',
          value: { language: 'eng', description: 'iTunPGAP', text: '0' } },
        { id: 'TENC', value: 'Music 13.10.3.1' },
        { id: 'COMM',
          value:
           { language: 'eng',
             description: 'iTunNORM',
             text:
              ' 000003B5 000003DA 00002F21 00003563 000023B6 0000D74E 00007267 000077A0 0000D7B6 0000D7B6' } },
        { id: 'COMM',
          value:
           { language: 'eng',
             description: 'iTunSMPB',
             text:
              ' 00000000 00000210 0000084C 00000000006E6BA4 00000000 00641054 00000000 00000000 00000000 00000000 00000000 00000000' } },
        { id: 'TSSE', value: 'Music 13.10.3.1' },
        { id: 'TXXX:HANDLER_NAME', value: 'Music 13.10.3.1' },
        { id: 'APIC',
          value:
           { format: 'image/jpeg',
             type: 'Other',
             description: '',
             data:
              <Buffer ff d8 ff e0 00 10 4a 46 49 46 00 01 01 00 00 48 00 48 00 00 ff e1 00 4c 45 78 69 66 00 00 4d 4d 00 2a 00 00 00 08 00 01 87 69 00 04 00 00 00 01 00 00 ... > } },
        { id: 'TDRC', value: '2020' },
        { id: 'TALB', value: 'Sorry You Couldn\'t Make It' },
        { id: 'TRCK', value: '8/10' },
        { id: 'TSRC', value: 'USJZA2032708' },
        { id: 'COMM',
          value: { language: 'eng', description: '', text: 'CLEAN' } },
        [length]: 14 ],
```

**Comment by rossgrady** [comment]
Oh huh so in ID3v2parser you have makeDescriptionTagName to handle these situations but I'm not sure it's firing. I will keep poking around.

**Comment by Borewit** [comment]
Fixed in [v6.4.0](https://github.com/Borewit/music-metadata/releases/tag/v6.4.0)
